Tasks/VsTest/distributedtest.ts

```diff
@@ -143,12 +143,32 @@ export class DistributedTest {
             throw new Error('Failed to start Modules/DTAExecutionHost.exe.');
         });
 
+        var consolidatedCiData = {
+                    agentFailure: false,
+                    batchingType: models.BatchingType[this.dtaTestConfig.batchingType],
+                    batchsize: this.dtaTestConfig.numberOfTestCasesPerSlice,
+                    codeCoverageEnabled: this.dtaTestConfig.codeCoverageEnabled,
+                    environmenturi: this.dtaTestConfig.dtaEnvironment.environmentUri, 
```

Comment:
you might actually want to add whether testplan/test assembly/on demand run scenario